client/src/includes/initTagField.ts

```diff
@@ -0,0 +1,34 @@
+// import  $ from 'jquery'
+/** 
+ *This is a function initTagField
+ *@function
+ *@param {string} id - The object id.
+ *@param {string} autocompleteUrl - The 
+ autoComplete Url.
+ *@param {object} options - Other options 
+ passed to initTagField
+ */
+function initTagField(id: string, autocompleteUrl: string, options: object) {
+  const tagFieldElement = document.getElementById(id)
+
+  if (!tagFieldElement) {
+    return ;
+  }
+    const finalOptions = {
+        autocomplete: { source: autocompleteUrl },
+        preprocessTag(val: any) {
+          // Double quote a tag if it contains a space
+          // and if it isn't already quoted.
+          if (val && val[0] !== '"' && val.indexOf(' ') > -1) {
+            return '"' + val + '"';
+          }
+  
+          return val;
+        },
+        ...options,
+      };
+    
+      // $('#' + id).tagit(finalOptions);
+    }
+
+    export { initTagField };
```

Comment:
For some reason, when I checked out your branch, this line was missing (the export) so not sure what happened.

Also, be sure to set your editor up correctly as it looks like your line feeds are incorrect. or at least run `npm run format` when finished editing and commit the output.